coq-doc.opam

```diff
@@ -27,7 +27,7 @@ build-env: [
 ]
 
 build: [
-  [ "dune" "build" "@refman" "-j" jobs ]
+  [ "dune" "build" "coq-doc.install" "-j" jobs ]
```

Comment:
This should be `"-p" "coq-doc"` , see upstream documentation https://dune.readthedocs.io/en/stable/opam.html